projects/iio_adpd1080/src/parameters.h

```diff
@@ -0,0 +1,68 @@
+/***************************************************************************//**
+ *   @file   iio_adpd1080/src/parameters.h
+ *   @brief  Parameters Definitions.
+ *   @author DBogdan (dragos.bogdan@analog.com)
```

Comment:
looks like you missed the name here

Reply:
I did not make this file. It is standard for these projects. Should we have only one that multiple projects import?